### DISCUSSION TITLE
阿蒂蜜斯計畫，睽違已久的登月計畫

### DISCUSSION THREAD

**Discussion by AnAnistired** [1101-GS4719-C/Python-Community-GS4719-C#714] [General]
在美蘇太空競賽NASA登月後，人類陸續有四次成功登陸月球，但在之後的五十幾年間卻不再有任何人類登月的紀錄，這次在各國合作的Artemis program下計畫最終在月球建設永久基地，最近會先把十顆cubesat送上去，雖然發射時間似乎又延期了，我怎麼又說「又」呢😂

https://www.space.com/nasa-artemis-1-moon-mission-cubesats
(https://www.space.com/nasa-artemis-1-moon-mission-cubesats)

**Comment by WEI-CHEN-CHUN** [comment]
Artemis 最新發射日期是11-14
希望不要再延後了
